x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -5,8 +5,12 @@
  * 2.0.
  */
 
-export const ENDPOINT_ACTIONS_INDEX = '.logs-endpoint.actions-default';
-export const ENDPOINT_ACTION_RESPONSES_INDEX = '.logs-endpoint.action.responses-default';
+/** endpoint data streams that are used for host isolation  */
+/** for index patterns `.logs-endpoint.actions-* and .logs-endpoint.action.responses-*`*/
+export const ENDPOINT_ACTIONS_DS = '.logs-endpoint.actions';
+export const ENDPOINT_ACTIONS_INDEX = `${ENDPOINT_ACTIONS_DS}-default`;
```

Comment:
nit: I am not a fan of this pattern. Unless `ENDPOINT_ACTIONS_DS` will be re-used somewhere else than here, it is easier to read the plain text and replacing this is a matter of a search/replace editor command.